test/unit/apiCallable.ts

```diff
@@ -364,6 +364,20 @@ describe('retryable', () => {
     });
   });
 
+  it('retry fails for exceeding total timeout', done => {
+    const spy = sinon.spy(fail);
+    const apiCall = createApiCall(spy, settings);
+    apiCall({}, undefined, err => {
+      assert.ok(err instanceof GoogleError);
+      assert.strictEqual(
+        err.message,
+        'Function  takes 100 milliseconds which exceeds retry total timeout 100 milliseconds before any response was received. The retry settings is {"initialRetryDelayMillis":0,"retryDelayMultiplier":0,"maxRetryDelayMillis":0,"initialRpcTimeoutMillis":0,"rpcTimeoutMultiplier":0,"maxRpcTimeoutMillis":0,"totalTimeoutMillis":100}.'
```

Comment:
The function name is missing here - is it a test setup problem, or a problem with `func.name` in the code? Will it actually work with the real function call?

Reply:
In general, this is a user facing error message, and I'm not sure we need to dump the whole JSON object here - maybe just the total timeout (the one that is compared to)?

Reply:
right, the name of function name is empty here, actually not sure the function name is helpful because users might just want to know which API they are calling.
re: timeout setting: I will just print out the total timeout value.